src/sentry/search/events/datasets/spans_metrics.py

```diff
@@ -805,6 +821,101 @@ def _resolve_rate(
             alias,
         )
 
+    def _resolve_regression_score(
+        self, args: Mapping[str, str | Column | SelectType | int | float], alias: str | None = None
+    ) -> SelectType:
+        return Function(
+            "minus",
+            [
+                Function(
+                    "multiply",
+                    [
+                        self._resolve_average_cond(args, None, "greater"),
+                        self._resolve_epm_cond(args, None, "greater"),
+                    ],
+                ),
+                Function(
+                    "multiply",
+                    [
+                        self._resolve_average_cond(args, None, "less"),
+                        self._resolve_epm_cond(args, None, "less"),
+                    ],
+                ),
+            ],
+            alias,
+        )
+
+    def _resolve_epm_cond(
+        self,
+        args: Mapping[str, str | Column | SelectType | int | float],
+        alias: str | None,
+        cond: str,
+    ) -> SelectType:
+        assert self.builder.params.start
+        assert self.builder.params.end
+        timestamp = cast(datetime, args["timestamp"])
+        if cond == "greater":
+            interval = (self.builder.params.end - timestamp).total_seconds()
+        elif cond == "less":
+            interval = (timestamp - self.builder.params.start).total_seconds()
+        else:
+            raise InvalidSearchQuery(f"Unsupported condition for epm: {cond}")
+
+        return Function(
+            "divide",
+            [
+                Function(
+                    "countIf",
+                    [
+                        Function(
+                            cond,
+                            [
+                                Column("timestamp"),
+                                timestamp,
+                            ],
+                        ),
+                    ],
+                ),
+                Function("divide", [interval, 60]),
+            ],
+            alias,
+        )
+
+    def _resolve_average_cond(
+        self,
+        args: Mapping[str, str | Column | SelectType | int | float],
+        alias: str | None,
+        cond: str,
+    ) -> SelectType:
+        conditional_aggregate = Function(
+            "avgIf",
+            [
+                Column("value"),
+                Function(
+                    "and",
+                    [
+                        Function(
+                            "equals",
+                            [
+                                Column("metric_id"),
+                                self.resolve_metric("span.self_time"),
```

Comment:
one of the args to the regression_score is a column, should this be that arg instead?